### PR TITLE
feat(TMG-1049): Switch commenting links based on environment, region

### PR DESCRIPTION
**Pull request by othetgyi**
### Description

This PR makes the link in the disabled-comment component a partial path so the article will show up on different environments. It also passes in the storefrontConfig as a prop whose value the join-the-conversation-dialog component can pick up from Render.

[TMG-1049](https://nidigitalsolutions.jira.com/browse/TMG-1049)

### Checklist

- [x] Have you done any manual testing?
- [ ] Does it have automated tests?
- [ ] Do you need any other PRs merged before this (if so please list)?
- [ ] Do you need to update the README/Runbook
- [ ] Have you checked for [accessibility](https://nidigitalsolutions.jira.com/wiki/spaces/TNLDigital/pages/3898048523/Accessibility+Dev+Guide#Checklist)?

### Screenshots (if appropriate):

Include screenshots if needed.

[TMG-1049]: https://nidigitalsolutions.jira.com/browse/TMG-1049?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/article-magazine-standard/src/article-magazine-standard.js [hunk 1]
```diff
@@ -70,7 +70,8 @@ class ArticlePage extends Component {
       commentingConfig,
       paidContentClassName,
       isPreview,
-      swgProductId
+      swgProductId,
+      storefrontConfig
```
Comment (lukejsikuade) [inline packages/article-magazine-standard/src/article-magazine-standard.js:74]: A lot of prop drilling, but this codebase has bigger problems

**Review by lukejsikuade — APPROVED** [review]
You

File: packages/article-comments/__tests__/web/__snapshots__/shared.web.test.js.snap [hunk 2]
```diff
@@ -135,7 +135,7 @@ exports[`User States RA Users 1`] = `
     </div>
     <a
       class="c3"
-      href="https://thetimes.co.uk/subscribe/digital?ILC=GB-TNL_The_Times-Conversion_Page-Homepage-2020"
+      href="[object Object]"
```
Comment (GiacN) [inline packages/article-comments/__tests__/web/__snapshots__/shared.web.test.js.snap:138]: should it be a readable string instead ? this suggest that a function is returning some sort of object

**Review by GiacN — COMMENTED** [review]
looking good to me, just a small comment

File: packages/article-comments/__tests__/web/__snapshots__/shared.web.test.js.snap (hunk 2 above)
Comment (othetgyi) [reply packages/article-comments/__tests__/web/__snapshots__/shared.web.test.js.snap]: Good point, I've fixed this test.

**Review by GiacN — APPROVED** [review]

**Pull request merged**